server/model/doctor.js

```diff
@@ -10,13 +11,68 @@ const doctorSchema = new mongoose.Schema({
         type: String,
         required: true
     },
+    /**Preferred language for effective patient-doctor communication and enhanced comprehension. */
+    Language: {
```

Comment:
- avoid using capital letters to start a variable, just change this. rest all seems great